lib/spoom/location.rb

```diff
@@ -15,42 +15,77 @@ class << self
       sig { params(location_string: String).returns(Location) }
       def from_string(location_string)
         file, rest = location_string.split(":", 2)
-        raise LocationError, "Invalid location string: #{location_string}" unless file && rest
+        raise LocationError, "Invalid location. File name is missing in location string: #{location_string}" unless file
```

Comment:
Let's reword a bit the error messages:

```suggestion
        raise LocationError, "Invalid location string `#{location_string}`: missing file name" unless file
```